internal/domain/helm/helm.go

```diff
@@ -263,3 +271,28 @@ func (h *Helm) SetState(st status.State) error {
 
 	return nil
 }
+
+func (h *Helm) SetHosts(vars variable.Variables) error {
+	if len(vars) == 0 {
+		return nil
+	}
+
+	hostExternalKey := fmt.Sprintf("QOVERY_HELM_Z%s_HOST_EXTERNAL", strings.ToUpper(strings.Split(h.ID.String(), "-")[0]))
+	hostInternalKey := fmt.Sprintf("QOVERY_HELM_Z%s_HOST_INTERNAL", strings.ToUpper(strings.Split(h.ID.String(), "-")[0]))
+
+	for _, v := range vars {
+		if v.Key == hostExternalKey {
+			h.ExternalHost = pointer.ToString(v.Value)
+			continue
+		}
+		if v.Key == hostInternalKey {
+			h.InternalHost = pointer.ToString(v.Value)
+			continue
+		}
+		if h.ExternalHost != nil && h.InternalHost != nil {
+			return nil
+		}
+	}
+
+	return ErrFailedToSetHosts
+}
```

Comment:
Looks like the this method doesn't return any error. 
The linter will complain iirc @ctjhoa ?